mslib/_tests/test_version.py

```diff
@@ -0,0 +1,11 @@
+import pytest

```

Comment:
have a look on our template https://github.com/Open-MSS/MSS/blob/develop/new_file_template.py
and use the header.